mllib/src/test/scala/org/apache/spark/mllib/tree/DecisionTreeSuite.scala

```diff
@@ -563,18 +136,10 @@ class DecisionTreeSuite extends SparkFunSuite with MLlibTestSparkContext {
     assert(!metadata.isUnordered(featureIndex = 0))
     assert(!metadata.isUnordered(featureIndex = 1))
 
-    val (splits, bins) = DecisionTree.findSplitsBins(rdd, metadata)
```

Comment:
I don't think we're losing any coverage.  The choice of splits should have nothing to do with the labels or the impurity.  There are already tests in spark.ml RandomForestSuite testing split calculations for ordered categorical features and for binary classification with continuous features, so those should cover the tests which were removed.